runtime/pkg/email/email.go

```diff
@@ -448,48 +472,57 @@ type TrialEndingSoon struct {
 	ToEmail      string
 	ToName       string
 	OrgName      string
+	UpgradeURL   string
 	TrialEndDate time.Time
 }
 
 func (c *Client) SendTrialEndingSoon(opts *TrialEndingSoon) error {
-	return c.SendInformational(&Informational{
-		ToEmail: opts.ToEmail,
-		ToName:  opts.ToName,
-		Subject: fmt.Sprintf("Your trial for %s is ending soon", opts.OrgName),
-		Title:   fmt.Sprintf("Your trial for %s is ending soon", opts.OrgName),
-		Body:    template.HTML(fmt.Sprintf("Your trial for %q is ending on %s. Upgrade to a paid plan to continue using Rill.", opts.OrgName, opts.TrialEndDate.Format("January 2, 2006"))),
+	return c.SendCallToAction(&CallToAction{
+		ToEmail:    opts.ToEmail,
+		ToName:     opts.ToName,
+		Subject:    fmt.Sprintf("Your trial for %s is ending soon", opts.OrgName),
+		Title:      fmt.Sprintf("Your trial for %s is ending soon", opts.OrgName),
+		Body:       template.HTML(fmt.Sprintf("Your trial for %q is ending on %s. Upgrade to a paid plan to continue using Rill.", opts.OrgName, opts.TrialEndDate.Format("January 2, 2006"))),
+		ButtonText: "Upgrade Now",
+		ButtonLink: opts.UpgradeURL,
 	})
 }
 
 type TrialEnded struct {
 	ToEmail            string
 	ToName             string
 	OrgName            string
+	UpgradeURL         string
 	GracePeriodEndDate time.Time
 }
 
 func (c *Client) SendTrialEnded(opts *TrialEnded) error {
-	return c.SendInformational(&Informational{
-		ToEmail: opts.ToEmail,
-		ToName:  opts.ToName,
-		Subject: fmt.Sprintf("Your trial for %s has ended", opts.OrgName),
-		Title:   fmt.Sprintf("Your trial for %s has ended", opts.OrgName),
-		Body:    template.HTML(fmt.Sprintf("Your trial for %q has ended. Your projects will be hibernated on %s. Upgrade to a paid plan to continue using Rill.", opts.OrgName, opts.GracePeriodEndDate.Format("January 2, 2006"))),
+	return c.SendCallToAction(&CallToAction{
+		ToEmail:    opts.ToEmail,
+		ToName:     opts.ToName,
+		Subject:    fmt.Sprintf("Your trial for %s has ended", opts.OrgName),
+		Title:      fmt.Sprintf("Your trial for %s has ended", opts.OrgName),
+		Body:       template.HTML(fmt.Sprintf("Your trial for %q has ended. Your projects will be hibernated on %s. Upgrade to a paid plan to continue using Rill.", opts.OrgName, opts.GracePeriodEndDate.Format("January 2, 2006"))),
+		ButtonText: "Upgrade to Team Plan",
```

Comment:
Can it not say the plan name and just send to the plan upgrade page which will pull the public plans and give option to choose one

Reply:
I am ok with not showing selector or anything just wanting to make sure that we are ok with saying `Team Plan` since we are hard-coding it and not fetching from plan. I think its ok for now.